pkg/leaderelection/leaderelection.go

```diff
@@ -0,0 +1,203 @@
+package leaderelection
+
+import (
+	"context"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/runtime"
+
```

Comment:
nit: remove the extra line breaks and run `make goimports`

Reply:
had run it, changing it manually